Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.yml

```diff
@@ -286,6 +295,16 @@ script:
       - Web
       required: false
       secret: false
+    - default: false
+      description: 'A json object of field_names and new_field_values for example
+        {"description": {"textarea": {"format": "html", "content": "The new description"}},
+        "name": {"text": "The new name"}}. Because of API limitations we currently
+        support only regular fields (We do not support fields like ''artifact'').
+        in case of conflicts refer to ''other-fields''.'
```

Comment:
This description is not very clear - how will the customer know what fields are "regular"? Can we prove a link or a list of allowed fields?
And what do you mean by "conflict"? When will this happen?